### PR TITLE
Thai/queryparam user

### LINKED ISSUE #130: [3] Query Parameter : User

### PR DESCRIPTION
**Pull request by athai**
Fixes #130. 

Tests will be uncommented when the branch is ready to be merged.

### CODE REVIEW

File: src/times.js [hunk 1]
```diff
@@ -134,6 +160,40 @@ module.exports = function(app) {
       const err = errors.errorServerError(error);
       return res.status(err.status).send(err);
     });
+
+    });
+
+    // CHECK OUT EVAN'S BRANCH TO UNDERSTAND HOW THIS SHOULD BE STRUCTURED
```
Comment (tschuy) [inline src/times.js]: What's with this big section of comment down here?

Comment (athai) [inline src/times.js]: It was a note to self... Forgot to delete it.

**Comment by Kennric** [comment]
This is looking good, I don't see any issues, but get a second opinion as well. +1

**Comment by tschuy** [comment]
uncomment the other tests.

**Comment by tschuy** [comment]
looks good other than the commented out tests. +1 once you uncomment them.

**Comment by tschuy** [comment]
+1, merge

**Pull request merged**